server/controllers/project.controller.js

```diff
@@ -261,7 +261,39 @@ function buildZip(project, req, res) {
     res.status(500).send({ error: err.message });
   });
 
-  res.attachment(`${project.name}.zip`);
+  function currentTime() {
```

Comment:
included in this project is a library called [moment](https://momentjs.com/) which makes it easy to convert times to a formatted string. i would prefer if you used this library as it converts this function to one or two lines of code.